tests/e2e-tests/config/setup.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+const {
+	setupSettings,
+	createTaxes,
+	createCoupons,
+	createProducts,
+	createShippingMethods,
+	enablePaymentGateways,
+} = require( './fixture-loaders' );
+const { setup: setupPuppeteer } = require( 'jest-environment-puppeteer' );
+
+module.exports = async ( globalConfig ) => {
+	await setupPuppeteer( globalConfig );
+	Promise.all( [
+		setupSettings(),
+		createTaxes(),
+		createCoupons(),
+		createProducts(),
+		createShippingMethods(),
+		enablePaymentGateways(),
+	] ).then( ( results ) => {
+		const [ , taxes, coupons, products, shippingMethods ] = results;
```

Comment:
``` 
const [ , taxes, coupons, products, shippingMethods ] = results;
```
we start destructuring with a comma, this means we want to ignore the first value in the array, that is `setupSettings` return value `undefined`